src/seedu/addressbook/data/person/SortPeople.java

```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.data.person;
+
+import java.util.Comparator;
+
+/**
+ *
+ * sort addressBook by Name in Alphebetical Order
+ */
+
+public class SortPeople implements Comparator<Person> {
```

Comment:
Can we implement similar interface on `Person` instead of creating a new class?

We can sort in the `UniquePersonList` instead.

Reply:
Will try and experiment of not implementing and creating a new class. thanks!